### DISCUSSION TITLE
AI Extension Autocomplete Endpoint not returning results

### DISCUSSION THREAD

**Discussion by sbacon94** [ueberdosis/tiptap#4691] [General Questions]
Hi There - I have installed the AI Extension and can see the POST request is being made to **https://api.tiptap.dev/v1/ai/text/autocomplete** but I never get a response. The request just hangs until it eventually times out. I've also tried the request in Postman passing the bearer token and x-app-id and I get the same result.

**Comment by svenadlung** [comment]
@sbacon94 could you please send an email to humans@tiptap.dev with more details (app id, email address of the team owner, etc.) if the issue still persists?
